### ISSUE TITLE
Media Writer on macOS prevents mounting

### ISSUE THREAD

**Issue by lbrabec** [FedoraQt/MediaWriter#713]
I noticed a strange behaviour on my MacBook Pro (Apple Silicon, M1 Pro), sometimes I couldn't open DMG files, sometimes USB flash drives wouldn't mount:

- opening DMG files results in an error alert "The disk image couldn't be opened, Failed to mount filesystems"
- plugged USB flash drives don't auto mount
- I can see the USB flash drive in Disk Utility.app, but mounting doesn't work (silently fails with no error)
- `hdiutil attach -verbose /dev/disk4s1` fails with error `DIHLDiskImageAttach() returned 112`, `Error 112 (no mountable file systems)`, this also happens to DMGs

However, the same DMGs mounted in macOS VM just fine and from other machines I knew that the USB flash drive was not damaged. Long session of googling resulted in nothing. But finally, after systematically checking what could be wrong, I found out that the reason was Fedora Media Writer.

When Fedora Media Writer app is running (I wrote a USB flash drive and forgot about the app window) it causes all above mentioned. Interestingly, when "About" dialog of Fedora Media Writer is opened, both DMGs and USB flash drives can be mounted (although it takes quite a while for it to happen).

Note:
As the workaround is easy, just quit the Fedora Media Writer, I'm writing this so verbosely primarily for others, so their googling is somewhat easier if they encounter this issue  🙂

Versions used:
macOS Sonoma 14.5 (23F79)
Fedora Media Writer 5.1.1

**Comment by ehjmx** [comment]
I was about to recommend Fedora Media Writer as a more privacy friendly alternative to balena etcher. It's also not as bloated and more straight forward if you just want to write an already downloaded img or iso than e.g. Raspberry Pi imager when I stumbled across this issue as well. 

Personally I can live with this behaviour and even use dd to write an image but our clients and customers wanting a gui it would be nice if this gets fixed. 

macOS 15.3.2
Fedora Media Writer 5.2.3

**Comment by lbrabec** [comment]
I don't think this is duplicate of #756, FMW while running (e.g. idling open on the initial screen) prevents _other_ DMGs (and USB flash drives) to be mounted. 

The issue, with the same error messages, is still present with versions:
MacOS Sequoia 15.4
Fedora Media Writer 5.2.3

**Comment by corneliusroemer** [comment]
Thanks for this issue, I hit a strange error flood when doing `brew update` - looks like Fedora Media Writer was to blame. Codex found your issue, thanks for making!
